frontend/src/components/Forms/UserProfile/UserProfileImage.tsx

```diff
@@ -31,7 +31,7 @@ const UserProfileImage: React.FC = () => {
         .then((res) => res.blob())
         .then((blob) => {
           imageURL = window.URL.createObjectURL(blob);
-          ImagesAPI.uploadMemberImage(blob);
+          ImagesAPI.uploadImage(blob, `image-signed-url/${userInfo ? userInfo.email : ''}`);
```

Comment:
we can move this prefix `image-signed-url/` into the ImagesAPI.uploadImage function